### PR TITLE
Translated using Weblate (Portuguese)

### PR DESCRIPTION
**Pull request by felipeerias**
Translation: Wolvic/Wolvic
Translate-URL: https://hosted.weblate.org/projects/wolvic/wolvic/pt/

### CODE REVIEW

File: app/src/main/res/values-pt/strings.xml [hunk 1]
```diff
@@ -126,7 +126,7 @@
     <string name="performance_unblock_page">Desbloquear página</string>
     <string name="performance_title">Desempenho ruim do site</string>
     <string name="performance_message">Tente reduzir o tamanho da janela para melhorar o desempenho. Desbloquear esta página pode travar a sua app.</string>
-    <string name="voice_samples_collect_data_dialog_title">Permitir que %1$s colete amostras de voz para pesquisa\?</string>
+    <string name="voice_samples_collect_data_dialog_title">Permitir que %1$s colete amostras de voz para pesquisa?</string>
```
Comment (pmatos) [inline app/src/main/res/values-pt/strings.xml:129]: s/colete/recolha/

colete is Brazilian Portuguese.

**Review by pmatos — CHANGES_REQUESTED** [review]
Besides the `colete` comment, lgtm.

File: app/src/main/res/values-pt/strings.xml (hunk 1 above)
Comment (svillar) [reply app/src/main/res/values-pt/strings.xml:129]: @felipeerias I guess it's better to change it in Weblate first and the resubmit the PR?

Comment (felipeerias) [reply app/src/main/res/values-pt/strings.xml:129]: @pmatos @svillar Yes, the translation appears like that in Weblate, better to change it there and I will update the PR.

Comment (felipeerias) [reply app/src/main/res/values-pt/strings.xml]: I have fixed the string in Weblate and brought the change into this PR.

Comment (pmatos) [reply app/src/main/res/values-pt/strings.xml]: lgtm

**Review by pmatos — APPROVED** [review]
lgtm

**Pull request merged**